public/video-ui/src/components/Atoms/AtomItem.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import {Link} from 'react-router';
+
+export default class AtomItem extends React.Component {
```

Comment:
Yep, agreed. I'll reformat this inside the redux PR as it'll cause a lot of conflicts/involve a lot of redundant work otherwise.